core/src/main/java/io/questdb/cairo/VarcharTypeDriver.java

```diff
@@ -490,6 +490,33 @@ public long getMinAuxVectorSize() {
         return 0;
     }
 
+    @Override
+    public long mergeShuffleColumnFromManyAddresses(
+            long indexFormat,
+            long primaryAddressList,
+            long secondaryAddressList,
+            long outPrimaryAddress,
+            long outSecondaryAddress,
+            long mergeIndex,
+            long destDataOffset,
+            long destDataSize
+    ) {
+        long rowCount = Vect.mergeShuffleVarcharColumnFromManyAddresses(
+                indexFormat,
+                primaryAddressList,
+                secondaryAddressList,
+                outPrimaryAddress,
+                outSecondaryAddress,
+                mergeIndex,
+                destDataOffset,
+                destDataSize
+        );
+        if (rowCount < 0) {
+            throw new IllegalStateException("unsupported column type shuffle result, returned row count=" + rowCount);
```

Comment:
it should be a CairoException - we don't handle exceptions uniformly, or at least i don't trust that we do